### PR TITLE
Fix build for layout

### PR DESCRIPTION
**Pull request by kseniaSs**
Fix npm run build:
Add .env.types
Remove layout from default.vue

### CODE REVIEW

File: src/vite-env.d.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
 // / <reference types="vite/client" />
+import 'vue-router';
```
Comment (slaveeks) [inline src/vite-env.d.ts:2]: Does it have to be imported here?

Comment (slaveeks) [reply src/vite-env.d.ts:2]: Maybe we can omit it?

File: src/vite-env.d.ts [hunk 2]
```diff
@@ -1,11 +1,14 @@
 // / <reference types="vite/client" />
+// To ensure it is treated as a module, add at least one `export` statement
+export {};
```
Comment (slaveeks) [inline src/vite-env.d.ts:3]: Can we omit it after removing module `import` statement?

Comment (kseniaSs) [reply src/vite-env.d.ts:3]: Without this {} , we get router types error, doesn't it?

```
src/application/router/index.ts:1:10 - error TS2305: Module '"vue-router"' has no exported member 'createRouter'.

1 import { createRouter, createWebHistory } from 'vue-router';
           ~~~~~~~~~~~~

src/application/router/index.ts:1:24 - error TS2305: Module '"vue-router"' has no exported member 'createWebHistory'.

1 import { createRouter, createWebHistory } from 'vue-router';
```

Comment (slaveeks) [reply src/vite-env.d.ts:3]: yes, you are right)

**Review by slaveeks — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**